Makefile

```diff
@@ -137,4 +137,7 @@ lint-fix:
 
 
 setup: requirements.txt
-	pip install -r requirements.txt
\ No newline at end of file
+	pip install -r requirements.txt
+
+generate-spec:
+	python -m scripts.generate_openapi
```

Comment:
Require the following env variables to work:
         export CONF=cloud-dev && \
	export PYTHONPATH=${PYTHONPATH} && \
	export SCHEMA_BUCKET_NAME=${PROJECT_ID}-europe-west2-schema && \
	export DATASET_BUCKET_NAME=${PROJECT_ID}-europe-west2-dataset && \
	export GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS} && \
	export AUTODELETE_DATASET_BUCKET_FILE=${AUTODELETE_DATASET_BUCKET_FILE} && \
	export LOG_LEVEL=${LOG_LEVEL} && \
	export PROJECT_ID=${PROJECT_ID} && \
	export PUBLISH_SCHEMA_TOPIC_ID=${PUBLISH_SCHEMA_TOPIC_ID} && \
	export PUBLISH_DATASET_TOPIC_ID=${PUBLISH_DATASET_TOPIC_ID} && \